Drivers/ManagePersonsPermissionCheckerTaskDisplayDriver.cs

```diff
@@ -5,17 +5,20 @@
 
 namespace Lombiq.TrainingDemo.Drivers;
 
-// ActivityDisplayDriver is specifically for implementing Workflow Activity Tasks.
-// Don't forget to register this class with the service provider (see: Startup.cs).
-public class ManagePersonsPermissionCheckerTaskDisplayDriver :
-    ActivityDisplayDriver<ManagePersonsPermissionCheckerTask, ManagePersonsPermissionCheckerTaskViewModel>
+// ActivityDisplayDriver is specifically for implementing workflow tasks. It performs a simple mapping of a ManagePersonsPermissionCheckerTask
+// to a ManagePersonsPermissionCheckerTaskViewModel and vice versa. Don't forget to register this class with the service provider (see: Startup.cs).
+public class ManagePersonsPermissionCheckerTaskDisplayDriver : ActivityDisplayDriver<ManagePersonsPermissionCheckerTask,
+    ManagePersonsPermissionCheckerTaskViewModel>
 {
-    protected override void EditActivity(
-        ManagePersonsPermissionCheckerTask activity,
-        ManagePersonsPermissionCheckerTaskViewModel model) => model.UserName = activity.UserName.Expression;
+    protected override void EditActivity(ManagePersonsPermissionCheckerTask activity, ManagePersonsPermissionCheckerTaskViewModel model) =>
+        model.UserName = activity.UserName.Expression;
 
-    protected override void UpdateActivity(
-        ManagePersonsPermissionCheckerTaskViewModel model,
-        ManagePersonsPermissionCheckerTask activity) => activity.UserName = new WorkflowExpression<string>(model.UserName);
+    protected override void UpdateActivity(ManagePersonsPermissionCheckerTaskViewModel model, ManagePersonsPermissionCheckerTask activity) =>
+        activity.UserName = new WorkflowExpression<string>(model.UserName);
```

Comment:
It was good that you had line breaks between arguments, just don't also start the method body on the same line (i.e. you need a line break after `=>`).